new/LOC7425BetaQweraTahtay.xml

```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7425BetaQweraTahtay" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Beta Qʷǝrāʿ Tāḥtāy</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-01-10">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="area">
+                    <placeName xml:lang="gez" xml:id="n1">ቤተ፡ ቍራዕ፡ ታሕታይ፡</placeName>
+                    <placeName xml:lang="gez" corresp="#n1" type="normalized">Beta Qʷǝrāʿ Tāḥtāy</placeName>
+                    <location><desc>Unknown place in the region of <placeName ref="LOC3832Hayq"/>.</desc></location>
```

Comment:
Could be the name of a church or a hamlet at the bottom of the Qwara' rock encoded as LOC7423QweraGadal

Reply:
Yes, I think it is very likely that these places are related. ቤተ፡ ቍራዕ፡ ታሕታይ፡ could be a descriptive explanation to explain the location of Zāygā Nǝśā (or Nǝśā alone) or it can be seen as a place by itself. In this case it can be seen and marked as a hamlet rather than an area. However, unless we have other information, it remains a hypothetical question whether it is likely to be a house, a farm or an area. In my opinion it is more likely to be a place name alone, because in a genitive construction it is more likely to be a fixed term. If one wanted to express the meaning "(not far) from the house, that is, under Qwǝrāʿ, it would be more reasonable to see እምቤት፡ ዘታሕተ፡ (ለ)ቍራዕ፡, but this is a semantic question and probably hypothetical. This seems to be the opinion of Taddesse Tamrat who gave the following translation. However, he interpreted Zāygā and Nǝśā as independent names and omitted the word tāḥtāy in his translation.

 
<img width="327" alt="Screenshot 2024_Beta Qwera Tahtay" src="https://github.com/BetaMasaheft/Places/assets/40291787/eb793271-330d-44c8-a462-c0567f73f865">

<img width="489" alt="Screenshot 2024_Qwira" src="https://github.com/BetaMasaheft/Places/assets/40291787/1a8d0914-90c8-4b57-b117-0c952ee10975">


```
<item xml:id="a14">
                           <locus target="#23v"/>
                           <desc type="LandGrant">Record of the assignment of land by  ʿaqqābe saʿat <persName ref="PRS14249Benyam"/> to 
                              <persName ref="PRS13198Tewodros"/>, dated to
                              <date notBefore="1431" notAfter="1432">6924 AM/year 84 of the 532-year cycle (= 1431/1432 A.D.)</date>. 
                              The mentioned king is probably <persName ref="PRS9176TaklaMa"/> or another petty king following
                              <persName ref="PRS10342Yeshaq"/> whose regnal name was possibly <foreign xml:lang="gez">ዳዊት፡</foreign>. 
                              The first seven lines were written with a different pen, but presumably by the same hand. The addition was quoted and 
                              translated by <bibl><ptr target="bm:Taddesse1970Abbots"/><citedRange unit="page">104-105</citedRange></bibl>.
                           </desc>
                           <q xml:lang="gez">በስመ፡ ሥሉስ፡ ቅዱስ፡ <gap reason="ellipsis" unit="lines" quantity="5"/> 
                              በ<date notBefore="1431" notAfter="1432">፷፻፡ በ፱፻፡ በ፳ወ፬፡ ዓመተ፡ ምሕረት፡ እምአዳም፡ በሐሳቦሙ፡ ለግብጻዊያን።</date>
                              ወ<date notBefore="1431" notAfter="1432">፹፬፡ ዓመት፡ ምሕረት፡ በሐሳብነ፡ ለኢትዮጵያዊያን፡</date>
                              በመዋዕሊሁ፡ ለንጉሥነ፡ <persName ref="PRS9176TaklaMa">ዳዊት፡</persName>
                              ወእንዘ፡ ሊቀ፡ ጳጳሳት፡ አባ፡ <persName ref="PRS5706JohnXI">ዮሐንስ፡</persName> 
                              ወጳጳስነ፡ አባ፡ <persName ref="PRS2469Bartalom">በርቶሎሜዎስ፡</persName> 
                              እንዘ፡ ዓቃቤ፡ ሰዓት፡ <persName ref="PRS14249Benyam">ብንያም፡ </persName>
                              ወእንዘ፡ ቂስ፡ ገበዝ፡ <persName ref="PRS14250Bakimos">በኪሞስ፡</persName> 
                              እንዘ፡ አፈ፡ ዓቃቤ፡ ሰዓት፡ <persName ref="PRS14251Tadewos">ታዴዎስ፡</persName> 
                              ወእንዘ፡ ሊቀ፡ ዲያቆናት፡ <persName ref="PRS14252Fiqtor">ፊቅጦር፡</persName>
                              እንዘ፡ እራቍማሰራ፡ <persName ref="PRS14253YebqeaLasyon">ይብቅዓ፡ ለጽዮን፡</persName> 
                              <gap reason="ellipsis" unit="lines" quantity="3"/>
                              ወሀብኩ፡ አነ፡ <persName ref="PRS14249Benyam">ብንያም፡</persName> ዓቃቤ፡ ሰዓት፡ 
                              ወልደ፡ <persName ref="PRS8538saraqaB">ሠረቀ፡ ብርሃን፡ </persName>
                              ለ<persName ref="PRS13198Tewodros">ቴዎድሮስ፡</persName> ካህን፡ ወመነኮስ፡ 
                              ምድረ፡ <del rend="effaced" unit="chars" quantity="2"/><placeName ref="LOC7423QweraGadal">ቍራዕ፡ ገደል።</placeName> 
                              ምድረ፡ <placeName ref="LOC7424ZaygaNesa">ዘይጋ፡ ንሣ፡</placeName> 
                              እም<placeName ref="LOC7425BetaQweraTahtay">ቤተ፡ ቍራዕ፡ ታሕታይ፡ </placeName>
                              ወምድረ፡ <placeName ref="LOC7426Dabaza">ደበዛ፡ </placeName>
                              ወምድረ፡ <placeName ref="LOC7427Ansalba">አንሰልባ፡</placeName> 
                              እምሀገረ፡ <placeName ref="LOC7428Yabasa">የ<del rend="effaced" unit="chars" quantity="1"/>በሳ፡</placeName>
                              <gap reason="ellipsis" unit="lines" quantity="3"/> ወተዝካሮሂ፡ ይግበሩ፡
                              <persName ref="PRS14254Langinos">ለንጊኖስ፡</persName> ወ<persName ref="PRS14255Luqyos ">ሉቅዮስ፡</persName> 
                              ውሉዱ፡ ወድኅረኒ፡ ውሉደ፡ ውሉዶሙ፡ እመቦ፡ <gap reason="ellipsis" unit="lines" quantity="10"/> ወበአፉሆሙ፡ ለኵሎሙ፡ ሊቃነ፡ መላእክት፡ 
                              ሥርወ፡ ወምቱረ፡ ይኩን፡ እምተስፋ፡ ሰማያዊያን፡ ወምድራዊያን፡ አሜን፡ ወአሜን፡ ለይኩን፡ ለይኩን።</q>
                        </item>
```